src/components/form/Form.tsx

```diff
@@ -129,16 +127,16 @@ const Form = () => {
 
         <FormButton
           type={"submit"}
-          color={Color.cream}
-          backgroundColor={Color.mediumGreen}
+          color={theme.colors.cream}
```

Comment:
I'd say in regular situations it's not convenient to pass those colors via props. 
It's not like the FormButton will have a lot of variants. 
It seems like you could have a regular Button component somewhere in your styles and then,if needed, extend it's styles in forms and other places with `const FromButton = styled(Button)```

Reply:
oh, right. Actually, in my case the generic Button component is called CallToAction. Would it be better to change its name to Button? Nevertheless, removed styles from here.